test/bootstrap_selects_test.rb

```diff
@@ -32,6 +32,20 @@ def setup
     assert_equivalent_xml expected, @builder.select(:status, [['activated', 1], ['blocked', 2]], { prompt: "Please Select" }, class: "my-select")
   end
 
+  test 'selects with addons are wrapped correctly' do
+    expected = <<-HTML.strip_heredoc
+      <div class="form-group">
+        <label class="control-label" for="user_status">Status</label>
+        <div class="input-group">
+          <span class="input-group-addon">Before</span>
+          <select class="form-control" id="user_status" name="user[status]"><option value="1">activated</option>
+      <option value="2">blocked</option></select>
+          <span class="input-group-addon">After</span>
+        </div>
```

Comment:
Yes, absolutely.  I added the </div> and structured the HTML example a little better as well.